packages/playground/src/utils/filter_farms.ts

```diff
@@ -0,0 +1,49 @@
+import type { InputFilterType } from "../types";
+import { isAlphanumeric, isInt, isNumeric, min, validateResourceMaxNumber } from "./validators";
+
+export type FilterFarmInputs = {
+  farmId: InputFilterType;
+  name: InputFilterType;
+  pricingPolicyId: InputFilterType;
+  totalIps: InputFilterType;
+};
+export const inputsInitializer: FilterFarmInputs = {
```

Comment:
The behavior of error messages in the filters is incorrect, for some cases when an incorrect value is entered it shows two messages saying `Failed to get farm` and it shows multiple errors in the console and in other cases as seen in the video below for the farms name filter it doesn't show anything.

[Screencast from 22-11-23 13:59:06.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/a5e934bb-8d64-405d-8b2d-e179b3343592)

Reply:
fixed, good catch

Reply:
The behavior of the error messages was fixed but the filtering itself is now broken.

[Screencast from 22-11-23 14:35:37.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/29494bf5-a93b-4fe0-9e2f-44557249dfbe)

Reply:
fixed, please check

Reply:
The original issue where the error message appeared twice + multiple errors in the console is back.

[Screencast from 22-11-23 15:30:38.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/8080c094-219f-4649-b83a-cfbbd50e7f40)